bin/tic_tac_toe.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+require_relative '../lib/game'
+require_relative '../lib/game_setting_setter'
+require_relative '../lib/game_setting/setting_types'
+
+include SettingTypes
+
+game_settings = GameSettingSetter.new()
+
+game = Game.new(
+    board_size: game_settings.settings[board_size],
+    player_2: game_settings.settings[opponent_type],
```

Comment:
I used to have player_1 as well, but it always defaults to Human. So I removed and only ask for board_size and player_2.

I was playing around with names like "Hero" and "Opponent" instead of Player 1 and Player 2 -- would that be more clear?

Reply:
Maybe? I was just thinking if you wanted a computer vs computer option, it might break down.